4.analyze-data/notebooks/UMAP/nbconverted/0.UMAP.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Extract UMAP embeddings for CFReT CP Features
+
+# In[1]:
+
+
+import glob
+import pathlib
+import pandas as pd
+import umap
+
+from pycytominer import feature_select
+from pycytominer.cyto_utils import infer_cp_features
+
+
+# In[2]:
+
+
+# Set constants
+umap_random_seed = 1234
```

Comment:
Why set the random seed to specifically 1234? I normally have seen this as 0, so I am curious the motivation.

Reply:
it is a random number :D